### PR TITLE
completed style for top champion section

### PR DESCRIPTION
**Pull request by dshcheung**

### CODE REVIEW

**Comment by dshcheung** [comment]
screen shot 2015-03-01 at 10 15 20 pm

current page

File: app/assets/stylesheets/summoner.css.scss [hunk 1]
```diff
@@ -71,29 +71,31 @@ div.summoner-page {
...
-          a {
-            background: white;
-            border-radius: 0;
+      div.secondary-nav {
+        ul.nav.nav-tabs.nav-justified{
+          li {
+            a {
+              background: white;
+              border-radius: 0;
+              border-bottom: 1px solid rgb(221, 221, 221);
```
Comment (h8rry) [inline app/assets/stylesheets/summoner.css.scss:80]: The color can be stored in a variable

File: app/controllers/summoners_controller.rb [hunk 2]
```diff
@@ -197,13 +197,18 @@ def update_stats_summary(id, region, summoner)
   end
 
   def get_stats_ranked
+    puts "start summoner"
     summoner = Summoner.find_by(summonerId: params['id'].to_i, region: params['region'].to_s)
-    if summoner[:stats_ranked].nil?
+    puts "after summoner"
+    if summoner.stats_rankeds.empty?
```
Comment (h8rry) [inline app/controllers/summoners_controller.rb:203]: The name `stats_rankeds` sounds a bit weird

File: app/assets/stylesheets/profileTabPartial.css.scss [hunk 3]
```diff
@@ -11,27 +11,37 @@ profile-tab > div.row {
   }
   div.league-summary {
     width: calc((100%/12*8) - 30px);
```
Comment (h8rry) [inline app/assets/stylesheets/profileTabPartial.css.scss:13]: Why need formulas?

**Pull request merged**